eclass/golang-base.eclass

```diff
@@ -23,7 +23,10 @@ if [[ -z ${_GOLANG_BASE} ]]; then
 
 _GOLANG_BASE=1
 
-DEPEND=">=dev-lang/go-1.4.2:="
```

Comment:
This will affect the entire tree. I guess the affected packages should be masked in the arch-specific package.mask instead.